test/chat_api_web/controllers/canned_response_controller_test.exs

```diff
@@ -33,6 +33,65 @@ defmodule ChatApiWeb.CannedResponseControllerTest do
       conn = get(authed_conn, Routes.canned_response_path(authed_conn, :index))
       assert json_response(conn, 200)["data"] == []
     end
+
+    test "returns unauthorized when auth is invalid", %{conn: conn} do
+      conn = get(conn, Routes.canned_response_path(conn, :index))
+
+      assert json_response(conn, 401)["errors"] != %{}
+    end
+
+    test "lists canned_responses only for this account",
+         %{authed_conn: authed_conn, account: account} do
+      insert(:canned_response, %{
+        name: "First response name",
+        content: "First response content",
+        account: account
+      })
+
+      conn = get(authed_conn, Routes.canned_response_path(authed_conn, :index))
+
+      assert json_response(conn, 200)["data"] |> length() == 1
+
+      # Add another account and canned response
+
+      another_account = insert(:account)
+
+      insert(:canned_response, %{
+        name: "Another canned response name",
+        content: "Another canned response content",
+        account: another_account
+      })
+
+      conn = get(authed_conn, Routes.canned_response_path(authed_conn, :index))
+
+      # Make sure we don't get the original canned_response when checking as the new user
+      assert json_response(conn, 200)["data"] |> length() == 1
+    end
+  end
+
+  describe "show canned_response" do
+    test "renders 404 when asking for another user's canned_response", %{
```

Comment:
do we have a test for the successful case for `show`?

Reply:
No, I can add that

Reply:
Right back atcha re: quick turnaround :D